logging/logging.go

```diff
@@ -40,9 +38,9 @@ const (
 )
 
 const (
-	LoggingFormatPlaintext = "plaintext"
-	LoggingFormatJSON      = "json"
-	LoggingFormatCEF       = "cef"
+	PlaintextFormatString = "plaintext"
```

Comment:
why renamed?

Reply:
Because IDE highlighted those constants for me as they start same as package name